src/components/DropZone/types.ts

```diff
@@ -1,6 +1,13 @@
 export type DropZoneEvent = DragEvent | React.ChangeEvent<HTMLInputElement>;
 
+export enum Size {
+  ExtraLarge = 500,
+  Large = 300,
+  Medium = 160,
+  Small = 100,
+}
```

Comment:
Nit: the other enum goes from small to extra large, so for someone reading through this, it could be a bit more difficult to compare them.